flexmeasures/data/scripts/cli_tasks/data_delete.py

```diff
@@ -16,13 +18,49 @@ def fm_delete_data():
     """FlexMeasures: Delete data."""
 
 
+@fm_delete_data.command("account")
+@with_appcontext
+@click.option("--id", type=int)
+@click.option(
+    "--force/--no-force", default=False, help="Skip warning about consequences."
+)
+def delete_account(id: int, force: bool):
+    """
+    Delete an account, including their users & data.
+    """
+    if not force:
+        prompt = "Delete account including generic assets, users and all their data?"
```

Comment:
To further mitigate the problem of human error, can we be more explicit about what is going to be deleted? For example, I'd suggest printing at least a list of corresponding users, and a list of generic assets incl. counts of their sensors and their beliefs data.

Reply:
I now list the users and the generic assets.